standard/expressions.md

```diff
@@ -3832,7 +3868,7 @@ Unless one of these conditions is true, a binding-time error occurs.
 >    {
 >       if (x == null)
 >       {
->           throw new ArgumentNullException();
+>           throw new System.ArgumentNullException();
```

Comment:
We should probably discuss what convention we want for explicit use of namespaces vs using directives.

Reply:
removed qualifier, and marked as "Incomplete"